### PR TITLE
Strip out more bloat from toolbox

### PR DESCRIPTION
**Pull request by NeilFraser**
Also removes three private accesses from workspaceSvg into toolbox tree.

The strategy for workspace.updateToolbox is no longer to clear and repopulate the toolbox, but rather to throw away the old toolbox and rebuild a new one.  This is simpler and more reliable.

This commit trims off another KB from the compiled code.

### CODE REVIEW

File: core/css.js [hunk 1]
```diff
@@ -136,7 +136,6 @@ Blockly.Css.CONTENT = [
 
   '.blocklyNonSelectable {',
     'user-select: none;',
-    '-moz-user-select: none;',
```
Comment (NeilFraser) [inline core/css.js]: FYI: The current version of Mozilla now supports `user-select` without the `-moz` prefix.  And previous versions of Mozilla started supporting `-webkit-user-select` (!).  So these lines aren't needed any more.

File: core/components/component.js [hunk 2]
```diff
@@ -77,42 +77,36 @@ Blockly.Component = function() {
...
-  this.childIndex_ = null;
+  this.childIndex_ = {};
 };
 
 
 /**
  * The default right to left value.
- * @type {?boolean}
- * @private
+ * @type {boolean}
```
Comment (samelhusseini) [inline core/components/component.js]: Should this be @package?

File: core/css.js (hunk 1 above)
Comment (samelhusseini) [reply core/css.js]: Should we hold off on this one until the next release? Firefox 68 still has 4% market share in September 2019 according to https://www.stetic.com/market-share/browser/

Comment (NeilFraser) [reply core/css.js]: Public support in Firefox for `-webkit-user-select` (which we are keeping) was added in version 49.  That's from 2016.
https://developer.mozilla.org/en-US/docs/Web/CSS/user-select

Additionally, the failure mode is benign, it's just the ability to select text that probably shouldn't be selectable.

File: core/components/component.js (hunk 2 above)
Comment (NeilFraser) [reply core/components/component.js]: Good call.  Done.

File: core/css.js (hunk 1 above)
Comment (samelhusseini) [reply core/css.js]: Sounds good.

File: core/toolbox.js [hunk 3]
```diff
@@ -198,15 +198,37 @@ Blockly.Toolbox.prototype.init = function() {
   this.config_['cleardotPath'] = workspace.options.pathToMedia + '1x1.gif';
   this.config_['cssCollapsedFolderIcon'] =
       'blocklyTreeIconClosed' + (workspace.RTL ? 'Rtl' : 'Ltr');
+  this.renderTree(workspace.options.languageTree);
+};
+
+/**
+ * Fill the toolbox with categories and blocks.
+ * @param {!Node} languageTree DOM tree of blocks.
+ */
```
Comment (samelhusseini) [inline core/toolbox.js]: Just checking if this is intended to be a public API?

Comment (NeilFraser) [reply core/toolbox.js]: Now @package

**Pull request merged**